extensions-core/kinesis-indexing-service/src/main/java/org/apache/druid/indexing/kinesis/KinesisIndexTaskTuningConfig.java

```diff
@@ -81,7 +80,7 @@ public KinesisIndexTaskTuningConfig(
       Long handoffConditionTimeout,
       Boolean resetOffsetAutomatically,
       Boolean skipSequenceNumberAvailabilityCheck,
-      Integer recordBufferSize,
+      @Nullable Integer recordBufferSizeBytes,
```

Comment:
Do you think it'd make sense to log a warning if the eliminated property is provided?

Reply:
good thought, will add.